### ISSUE TITLE
Method to advance the queue

### ISSUE THREAD

**Issue by ianjackson8** [WVU-CS230-2023-01-Group10/GotNext#77]
Once a game is finished, there needs to be a method to advance the queue (dequeue the front team)
This event could be triggered by the user or the current team?

**Comment by ianjackson8** [comment]
On hold until #94 closes

**Comment by nathanmullins11** [comment]
@nathanmullins11 completed issue #77. The queue class is not yet implemented but the current queue will advance when a game has ended. A game is ended by both current players exiting the game.
